Submission burning questions

LevFendi opened the pull request:
<!--
Thank you for your submission!
-->

### Author info
Name: Sir Fendi
Suggested release date: May 6th

#### Requirements for media in submissions:
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] Images are hosted on Media server <!--(recommended width:896px - height:504px)-->
- [ ] Gifs are hosted on Media server
  - [ ] Gifs are [compressed](https://www.iloveimg.com/compress-image/compress-gif) <!--(recommended width:896px - height:504px)-->

#### For more detailed guidance and a full list of possible embed formats see: https://alt-f4.blog/ALTF4-X/
<!--_NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open._-->

LevFendi committed:
Burning Questions - initial article submission

LevFendi committed:
Initial submission comments:
*by Sir Fendi*

*version 1*

*Diagrams are included in a separate powerpoint file, conclusion will be added, calculations will be double-checked.*

*All feedback is welcome!*

LevFendi committed:
Revisions

LevFendi committed:
added preview image of the table

LevFendi committed:
More revisions

LevFendi committed:
Overhaul, intermediate edits

LevFendi committed:
Renamed file, overhaul is still WIP

LevFendi committed:
Overhaul WIP, image formating

LevFendi committed:
Overhaul done, last bits left to add

LevFendi committed:
Now ready for first review

LevFendi committed:
Wording improvements

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+. . . . . . . .
+
+Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to sci-fi level nuclear fuel. The most popular burner fuel is coal, which is the backbone of early game automation and it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
```
Nuclear fuel doesn't make me think of sci-fi since we have had nuclear reactors IRL for decades, although if you asked someone 50/100 years ago they'd definitely agree. The portable fusion reactor the player can put in their armor is definitely sci-fi, though. Or maybe you're calling it sci-fi because the player can have hundreds of them in their inventory or some other aspect of it that isn't realistic?

LevFendi replied:
Nuclear rocket fuel is mostly experimental IRL but yeah it is closer to reality than sci-fi, good point.

MyNameIsTrez replied:
> up to sci-fi level nuclear fuel

Are you not referring to the nuclear fuel that goes into nuclear reactors here? The reactor falls under "burner device". If you meant nuclear rocket fuel I'd change that quote to that.

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+%}
+
+We see that while uranium fuel cells can only be used inside nuclear reactors, they produce energy on the order of GJ instead of MJ, and the energy yield is not simply boosted but multiplied via the reactor neighbor bonus. Hence, up to hundreds of times more energy is gained from the same piece of uranium-235 if used in a nuclear reactor instead of nuclear fuel. This would suggest that if one has ...
```
> Hence, up to hundreds of times more energy is gained

Maybe show a quick calculation for this using 0, 2 and 3 neighbors (1 neighbor isn't seen often and 4 neighbors of course prevents automatic fuel insertion and extraction).

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+* **Gains in energy:** If you get more energy out of the processed fuel, then your demand for the unprocessed fuel decreases and so your resource patches run out more slowly. We determined the energy gains for different fuels earlier.
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses.
+- **Gains in energy density (per stack and per unit):** More energy per stack means more vehicle range per slot of fuel, which makes a big difference for trains because of their limited fuel slots. Meanwhile, more energy per unit/stack can be interpreted as fewer units/stacks per energy, which means less logistics are required to move the same amount of energy. Needing fewer bots, belts, and/or t ...
```
> which makes a big difference for trains because of their limited fuel slots

I'm assuming you're talking about the fuel slots of the Locomotive and not the Cargo wagons here, cause for Cargo wagons this point makes sense but for Locomotives their fuel use should cost an insignificant amount of energy to be transported.

Most bases automatically insert fuel at the unloading stations, so I don't think it matters how much fuel Locomotives can hold since in my experience they can travel a massive distance before running out of fuel, even if it is coal.

How much fuel can be held matters a little bit more for cars and tanks maybe, but they don't use that much fuel, and I personally manage to remember that they have to be refueled when on a journey by bringing some fuel in my/their inventory.

LevFendi replied:
I did originally mean locomotive slots. Good point that they can travel massive distances even with coal as long as there is a reasonable setup for refueling. I guess this energy density advantage does not matter so much. Ditto about cars and tanks.

The point about wagons I made later in the paragraph but I can explain it better.

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+This article started with a little burning question that brought more burning questions and evolved into an exploration of all of Factorio's fuels. We uncovered the following:
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 1.57 times as much energy per coal.
```
```suggestion
**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 1.57 times as much energy per coal.
```
You previously said

> So we see that simply liquifying the coal gets us about 57% more energy out of it.

So I'd either turn this "1.57 times as much energy" into "57% more energy" or the other way around for consistency. I personally find 57% clearer.

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 1.57 times as much energy per coal.
+
+**(2) Coal liquification with cracking:** If we also crack the heavy oil before making the solid fuel, it yields 2.03 times as much energy per coal.
```
Same comment as above with either saying "2.03 times as much energy" or "103% more energy".

Firerazer commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+| ![Assembling machine 2.png](https://wiki.factorio.com/images/thumb/Assembling_machine_2.png/32px-Assembling_machine_2.png) Assembling machine 2 | 0.150*                         | ![Rocket fuel.png](https://wiki.factorio.com/images/thumb/Rocket_fuel.png/32px-Rocket_fuel.png)![Uranium fuel cell.png](https://wiki.factorio.com/images/thumb/Uranium_fuel_cell.png/32px-Uranium_fuel_cell.png)           ...
+| ![Centrifuge.png](https://wiki.factorio.com/images/thumb/Centrifuge.png/32px-Centrifuge.png) Centrifuge                                         | 0.350                          | ![Nuclear fuel.png](https://wiki.factorio.com/images/thumb/Nuclear_fuel.png/32px-Nuclear_fuel.png)                                                                                                                         ...
+
+*If we use assembling machine 3 instead, the power use would be 0.375MW and crafting speed 1.25.
```
ok, and so what? is there any meaningful impact, or it is only a scaling problem with no other side effect?

LevFendi replied:
I assume you mean the footnote? It is meant to acknowledge that the value of E is different if you select the other assembler tier. I should explain that better.

Firerazer replied:
yes, it is the *if we... part that, for me, need way more explanations on the impact or just have to go ;)

Firerazer commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+*If we use assembling machine 3 instead, the power use would be 0.375MW and crafting speed 1.25.
+
+The total processing energy ("E") needed for a conversion can be calculated using the machine power use ("P") multiplied by the total crafting time for all of the input ("T"). This total crafting time is the duration of one crafting cycle ("d") divided by the machine crafting speed ("s"), multiplied by the number of crafting cycles ("c") to cover the entire input quantity. Hence we summarize proc ...
```
Time is usually a little "t", not its big version "T" in physics. I recommend to switch for the little "t" for all things related to time in equations

Firerazer commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,272 @@
+
+`E = P * T = P * d * c / s`
+
+While we calculate these values without modules to see the "default" situation, I would highly recommend to fill these machines with at least efficiency 1 modules in practice, as it would reduce E by 60%-80%. Other module combinations can offer even better results, like extra yield from speed-productivity synergy. [This guide]([Tutorial:Producing power from oil - Factorio Wiki](https://wiki.facto ...
```
is it at this moment that you should discuss about adding modules? because modules could/should be placed into all machines, being efficience/productivity or speed, having impact on the whole chain and maybe reversing some conclusions? then, if this is true, I would remove all module from the initial calculation, and discuss them into a dedicated section, with maybe "the best module setup to maximise <watéver> from the chain"

LevFendi replied:
I agree that talking about modules right after emphasizing that they are not used is unnecessary. I'll leave this discussion to only the end. 

I had not thought that they might be effective enough to reverse the conclusions, but now that you mention it, I realized that one can indeed increase rocket fuel yield by 40% with productivity modules and that would probably reverse that conclusion. I think I'll look into it a bit more and rewrite some conclusions...

LevFendi committed:
1.2 ready, needs one last pass

LevFendi committed:
v1.2 now ready

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+In the mid game, solid fuel is introduced as the first processed fuel, made initially from crude oil. Conveniently, solid fuel is used to later make rocket fuel, which is in turn used to make nuclear fuel. These processed fuels offer noticeable advantages over coal, like having greater energy density both per unit and per stack, and an acceleration bonus when used in vehicles. More curiously, how ...
```
```suggestion
In the mid-game, solid fuel is introduced as the first processed fuel, made initially from crude oil. Conveniently, solid fuel is used to later make rocket fuel, which is in turn used to make nuclear fuel. These processed fuels offer noticeable advantages over coal, like having greater energy density both per unit and per stack, and an acceleration bonus when used in vehicles. More curiously, however, the late-game also brings the technology to liquify good old coal and convert it into solid fuel. This raised for me the question: Does processing coal into solid fuel get more energy out of it? In other words, if you liquify 10 units of coal, make solid fuel out of the oils and gas that it yields, and then burn the solid fuel, does it provide more energy than directly burning the coal, even with all that processing? And what if you further processed it into rocket fuel? Similar burning questions followed and snowballed into this general exploration of the energy contents of different burner fuels as well as the energy costs of converting between them.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+. . . . . . . .
+
+Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to experimental and extreme nuclear rocket fuel. The most popular burner fuel is coal, which is the backbone of early game automation and it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
```
```suggestion
Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to experimental and extreme nuclear rocket fuel. The most popular burner fuel is coal, which is the backbone of early-game automation; it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+In the mid game, solid fuel is introduced as the first processed fuel, made initially from crude oil. Conveniently, solid fuel is used to later make rocket fuel, which is in turn used to make nuclear fuel. These processed fuels offer noticeable advantages over coal, like having greater energy density both per unit and per stack, and an acceleration bonus when used in vehicles. More curiously, how ...
+
+As I set off to answer these questions, a great info source was the [the wiki page for fuels](https://wiki.factorio.com/Fuel) and I approached the conversions in terms of energy produced and consumed per unit of input, rather than the production rates per building. At first I tried solving many equations with a growing list of variables, but it was much easier to convert everything into diagrams, ...
```
```suggestion
As I set off to answer these questions, a great info source was [the wiki page for fuels](https://wiki.factorio.com/Fuel) and I approached the conversions in terms of energy produced and consumed per unit of input, rather than the production rates per building. At first I tried solving many equations with a growing list of variables, but it was much easier to convert everything into diagrams, which now summarize everything nicely. Hence, in this piece, I present some questions about different burner fuels and answer them mainly using conversion diagrams. The results provide some interesting tidbits about managing your resources and they help to complete the big picture for burner fuels.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+## Making Conversion Diagrams
+
+In the diagrams, the recipes for various fuel-related crafting processes are applied. The input and output ratios are preserved but we run each process fractionally to match the input quantities rather than aiming for full crafting cycles. Most of the used recipes can be checked quickly from [the wiki page for oil processing](https://wiki.factorio.com/Oil_processing). The energy values of the fue ...
```
```suggestion
In the diagrams, the recipes for various fuel-related crafting processes are applied. The input and output ratios are preserved, but we run each process fractionally to match the input quantities, rather than aiming for full crafting cycles. Most of the used recipes can be checked quickly from [the wiki page for oil processing](https://wiki.factorio.com/Oil_processing). The energy values of the fuels are in-game constants, referenced in [the wiki page for fuels](https://wiki.factorio.com/Fuel). The energy consumed to heat steam to 165°C is referenced in [the wiki page for steam](https://wiki.factorio.com/Steam). 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+In the diagrams, the recipes for various fuel-related crafting processes are applied. The input and output ratios are preserved but we run each process fractionally to match the input quantities rather than aiming for full crafting cycles. Most of the used recipes can be checked quickly from [the wiki page for oil processing](https://wiki.factorio.com/Oil_processing). The energy values of the fue ...
+
+As for the electricity consumption during the fuel processing, it depends on the machine used and number of crafting cycles needed. Calculations are simplified a lot thanks to the machines using the same amount of power per second for each of their processes. These power uses are summarized in the table:
```
```suggestion
As for the electricity consumption during the fuel processing, it depends on the used machines and the number of crafting cycles they need. Calculations are simplified a lot thanks to the machines using the same amount of power per second for each of their processes. These power uses are summarized in the table:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+## Adding cracking to coal liquification
+
+Cracking adds and extra step to the process, but looking at the recipes shows that it can increase solid fuel yield: Cracking heavy oil provides 30/40 or 75% as much light oil. Meanwhile, 10 light oil is enough to make 1 solid fuel rather than 20 heavy oil, meaning that it has 200% of the yield. Combining these, by cracking heavy oil we get 75% * 200% = 150% as much solid fuel as before. 
```
```suggestion
Cracking adds an extra step to the process, but looking at the recipes shows that it can increase solid fuel yield: Cracking heavy oil provides 30/40 or 75% as much light oil. Meanwhile, 10 light oil is enough to make one solid fuel, rather than 20 heavy oil, meaning that it has 200% of the yield. Combining these, by cracking heavy oil we get 75% * 200% = 150% as much solid fuel as before. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+Cracking adds and extra step to the process, but looking at the recipes shows that it can increase solid fuel yield: Cracking heavy oil provides 30/40 or 75% as much light oil. Meanwhile, 10 light oil is enough to make 1 solid fuel rather than 20 heavy oil, meaning that it has 200% of the yield. Combining these, by cracking heavy oil we get 75% * 200% = 150% as much solid fuel as before. 
+
+However, the same cannot be said about cracking light oil. It provides 20/30 or about 66.7% as much petroleum gas, but also 20 petroleum gas is required for making solid fuel instead of 10 light oil, meaning 50% as much yield. Combining these, by cracking light oil we get 66.7% * 50% = 33.4% as much solid fuel as before, meaning that light oil cracking should be avoided.
```
```suggestion
However, the same cannot be said about cracking light oil. It provides 20/30 or about 66.7% as much petroleum gas, but 20 petroleum gas is also required for making solid fuel, instead of 10 light oil, meaning 50% as much yield. Combining these, by cracking light oil we get 66.7% * 50% = 33.4% as much solid fuel as before, meaning that light oil cracking should be avoided.
```

LevFendi replied:
"but also the recipe for solid fuel requires 20 petroleum gas rather than 10 light oil"

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We see that we now get 7.375 solid fuel from the same 10 coal instead of 5.75. That seems great but the cracking also consumes additional energy. So, let's look at total energy gain:
```
```suggestion
We see that we now get 7.375 solid fuel from the same 10 coal, instead of 5.75. That seems great, but the cracking also consumes additional energy. So, let's look at total energy gain:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+`Percent gain = 41.12MJ / 40MJ = 102.8% `
+
+We find that liquifying coal and including cracking leads getting about **twice its original energy**. In other words, if you liquify coal before burning it, you only need half as much as you did before! While this suggests that liquifying coal on a large scale is a great idea, other factors like setup costs need to be considered before jumping into it, as discussed later. 
```
```suggestion
We find that liquifying coal and including cracking leads to getting about **twice the coal its original energy**. In other words, if you liquify coal before burning it, you only need half as much as you did before! While this suggests that liquifying coal on a large scale is a great idea, other factors like setup costs need to be considered before jumping into it, as discussed later. 
```

LevFendi replied:
I'm not sure about the added  "the coal" here...

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We see that advanced oil processing yields 8.5 solid fuel from the same 100 oil. That's nearly 4 times the yield for basic oil processing! However we can do better: Once again, cracking the heavy oil would offer us some more energy:
```
```suggestion
We see that advanced oil processing yields 8.5 solid fuel from the same 100 oil. That's nearly four times the yield for basic oil processing! However, we can do better: Once again, cracking the heavy oil would offer us some more energy:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+* **Exclusive uses:** While both coal and crude oil can be used to make oil products, the recipes that use coal exclusively (plastic, explosives, capsules) greatly outweigh those for crude oil (flamethrower ammo). 
+
+Considering all these factors together, it seems to generally be better to save most of your coal for plastic/explosive production and use mostly oil for solid fuel production, unless coal/oil abundances encourage otherwise. Thus, we have solid fuel production fully covered but is not the end of the fuel discussion, as we can make into rocket fuel.
```
```suggestion
Considering all these factors together, it seems to generally be better to save most of your coal for plastic/explosives production and to use mostly oil for solid fuel production, unless coal/oil abundances encourage otherwise. Thus, we have solid fuel production fully covered, but this is not the end of the fuel discussion, as we can still turn it into rocket fuel.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We see that a single unit rocket fuel contains 100MJ of energy while the required ingredients of 10 solid fuel already holds 120MJ, and the 10 light oil can be made into an 11th solid fuel to provide almost 12MJ more. Hence, we see that processing solid fuel into rocket fuel makes you lose energy, before even accounting for processing energy. However, as we will cover later on, rocket fuel has ad ...
```
```suggestion
We see that a single unit of rocket fuel contains 100 MJ of energy, while the required ingredients of 10 solid fuel already holds 120 MJ, and the 10 light oil can be made into an 11th solid fuel to provide almost 12 MJ more. Hence, we see that processing solid fuel into rocket fuel makes you lose energy, before even accounting for processing energy. However, as we will cover later on, rocket fuel has advantages over solid fuel that may outweigh the energy losses. Furthermore, it is a necessary ingredient if we want to make nuclear fuel, which may increase the overall energy gains, despite the losses at this stage.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+## Going nuclear
+
+Nuclear fuel is still pretty experimental in the real world, but Factorio offers a standardized variant that is made from regular rocket fuel and uranium-235 and it presents itself as the ultimate burner fuel. Like with our examination of rocket fuel, the recipe itself is enough to determine energy gains:
```
```suggestion
Nuclear fuel is still pretty experimental in the real world, but Factorio offers a standardized variant that is made from regular rocket fuel and uranium-235 and presents itself as the ultimate burner fuel. Like with our examination of rocket fuel, the recipe itself is enough to determine energy gains:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We see that nuclear fuel offers an impressive energy output of 1.21GJ, which is [a reference](https://wiki.factorio.com/Nuclear_fuel), but also around 10 times as much energy as the solid fuel first used to make the rocket fuel. This means that if you were to switch from burning coal directly to making it into nuclear fuel to burn instead, your coal demand would drop to a tiny fraction, to as lit ...
```
```suggestion
We see that nuclear fuel offers an impressive energy output of 1.21GJ, which is [a reference](https://wiki.factorio.com/Nuclear_fuel), but also around 10 times as much energy as the solid fuel first used to make the rocket fuel. This means that if you were to switch from burning coal directly to making it into nuclear fuel to burn instead, your coal demand would drop to a tiny fraction, to as little as 1/20th of what it was before. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+## Making the most of your uranium
+
+Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation: They can only be used inside nuclear reactors. One fuel cell provides one reactor a constant supply of 40MW of heat for exactly 200 seconds, which adds up to a total en ...
```
```suggestion
Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation: They can only be used inside nuclear reactors. One fuel cell provides one reactor a constant supply of 40 MW of heat for exactly 200 seconds, which adds up to a total energy of `40 MW * 200s = 8000 MJ = 8 GJ`. If the reactor has any neighbors, the neighbor bonus kicks in, and the bonus from each neighbor leads to an additional 40MW of heat being supplied by the same fuel cell. Since the reactor can have up to four neighbors, the fuel cell can provide up to five times its original yield, which is `8 GJ * 5 = 40 GJ`. We are already dealing with GJ instead of MJ when discussing a single fuel cell, and on top of that its recipe yields 10 of them per piece of uranium-235:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Hence, we see that we can get up to hundreds of times more energy from the same piece of uranium-235 if it is used in a nuclear reactor instead of nuclear fuel. This would suggest that if one has a uranium-235 supply already, they should just switch to nuclear power and electric machines instead of burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex En ...
```
```suggestion
Hence, we see that we can get up to hundreds of times more energy from the same piece of uranium-235 if it is turned into uranium fuel cells, instead of being turned into nuclear fuel. This would suggest that if one already has a uranium-235 supply, they should just switch to nuclear power and electric machines, instead of using burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex Enrichment, which allows for the easy production of uranium-235. This means that using uranium efficiently stops being an issue around the time you unlock nuclear fuel.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+Here is what the modules do:
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules because they are quite cheap to make and they cut down power usage by 60% or even 80% (the effi ...
```
```suggestion
- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules, because they are quite cheap to make and they cut down power usage by 60% or even 80% (the efficiency limit), depending on how many module slots the machines have.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules because they are quite cheap to make and they cut down power usage by 60% or even 80% (the effi ...
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speeds of machines ("C"), which as seen from the equation, reduces the power usage. For every speed module, the increase in P is greater than the increase in C, wh ...
```
```suggestion
- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation, reduces the power usage. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which results in decreased energy gains overall. However, speed modules can be used to mitigate the slowdown from productivity modules, which might provide energy gains overall.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speeds of machines ("C"), which as seen from the equation, reduces the power usage. For every speed module, the increase in P is greater than the increase in C, wh ...
+
+- **Productivity modules:** They have the most interesting effects. They increase power usage ("P") and decrease crafting speed ("C"), both of which lead to an increase in E, which decreases the overall energy gains. However, productivity modules also increase the amount of processed fuel produced. Furthermore, if we keep using them in every step of the processing, we can get more and more fuel a ...
```
```suggestion
- **Productivity modules:** They have the most interesting effects. They increase power usage ("P") and decrease crafting speed ("C"), both of which lead to an increase in E, which decreases the overall energy gains. On the other hand, productivity modules increase the amount of processed fuel that is produced. Furthermore, this increase is compounded on every step of processing, which leads to exponential increases in the fuel output overall! The energy gains from the extra fuel might outweigh all the extra power consumption.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Now let's demonstrate the use of modules. First, let's look at the case of efficiency in coal liquification with heavy oil cracking. Refineries and chemical plants have 3 module slots, so we can place 3 efficiency 1 modules to reduce power consumptions by 80% for each: 
```
```suggestion
Now let's demonstrate the use of modules. First, let's look at the case of efficiency modules in coal liquification with heavy oil cracking. Refineries and chemical plants have three module slots, so we can place three efficiency 1 modules to reduce power consumptions by 80% for each machine:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to the energy gains of 116%, compared to the default case of 103%. This increase is nice but it is not that large because processing energy was already relatively small. Now let us look at the same example with 3 productivity 3 modules instead to see the effects of maximum productivity:
```
```suggestion
The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to an energy gain of 116%, compared to the default case of 103%. This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Now, let us look at the same example with three productivity 3 modules instead to see the effects of maximum productivity:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100MJ! However, we also see that total power consumption has greatly increased to about 50.6MJ. Nevertheless, our overall energy gains have become `185.6 - 50.6 - 40 - 1.5 = 93.5MJ`, which is about 134 ...
```
```suggestion
We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100 MJ! However, we also see that total power consumption has greatly increased to about 50.6 MJ. Nevertheless, our overall energy gains have become `185.6 - 50.6 - 40 - 1.5 = 93.5 MJ`, which is about 134% of energy gains, instead of the default case of 103%. Meanwhile, our machines are working at half the speed as before, which makes power get consumed for twice as long.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+We now find numbers somewhere in the middle of the previous two cases. We have about 12.2 solid fuel now, which is a good gain and it gives about 146.4MJ. We see that the total power consumption is now about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains instead of the default case of 103%. We discover that full productivi ...
```
```suggestion
We now find numbers somewhere in the middle of the previous two cases. We get about 12.2 solid fuel now, which is a good gain, giving us about 146.4MJ. We see that the total power consumption is about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains, instead of the default case of 103%. We discover that full productivity performs better in this case, despite the machines slowing down so much.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100MJ! However, we also see that total power consumption has greatly increased to about 50.6MJ. Nevertheless, our overall energy gains have become `185.6 - 50.6 - 40 - 1.5 = 93.5MJ`, which is about 134 ...
+
+Hence finally, let's try out the compensating effects of speed modules by switching to 2 productivity 3 modules and 1 speed 3 module each:
```
```suggestion
Now, let's try out the compensating effects of speed modules by switching to two productivity 3 modules and one speed 3 module for each machine:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+We now find numbers somewhere in the middle of the previous two cases. We have about 12.2 solid fuel now, which is a good gain and it gives about 146.4MJ. We see that the total power consumption is now about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains instead of the default case of 103%. We discover that full productivi ...
+
+Seeing the strong effects of productivity modules, I wonder if we can make rocket fuel production yield net energy gains too, so let's fill the assembling machine 2 productivity 3 modules:
```
```suggestion
Seeing the strong effects of productivity modules, I wonder if we can make rocket fuel production yield net energy gains too, so let's fill the assembling machine 2 with productivity 3 modules:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel at maximum productivity AM2'
```
```suggestion
 alt='Processing solid fuel into rocket fuel in an assembling machine 2 at maximum productivity'
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel at maximum productivity AM2'
+ caption='Processing solid fuel into rocket fuel at maximum productivity AM2'
```
```suggestion
 caption='Processing solid fuel into rocket fuel in an assembling machine 2 at maximum productivity'
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Well, the input and output have the same energy now, but the processing energy has grown big enough to wipe all the fuel energy gains. Looking at the cause of this high power cost, we see that it is because this recipe takes 30 seconds to craft while before we were working with 2 or 5 seconds. This difference means that a higher power consumption will has a much greater impact overall.
```
```suggestion
Well, the input energy cost and output energy generation are equal now, but the processing energy cost has grown big enough to wipe all the fuel energy gains. Looking at the cause of this high power cost, we see that it is because this recipe takes 30 seconds to craft, while before we were working with two or five seconds. This difference means that a higher power consumption will have a much greater impact overall.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Now we see that we can get a maximum of 9.125 solid fuel per 100 oil. We can now compare it with the maximum of 7.375 solid fuel from 10 coal. If we crunch these numbers, we find that 1 unit of coal produces about 8.08 times as much solid fuel as 1 unit of oil. 
```
```suggestion
Now we see that we can get a maximum of 9.125 solid fuel per 100 oil. We can now compare it with the maximum of 7.375 solid fuel from 10 coal. If we crunch these numbers, we find that one unit of coal produces about 8.08 times as much solid fuel as one unit of oil. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+`oil energy output = 1.03305MJ / oil`
+
+We find that 1 unit of coal holds roughly 8 times the energy as 1 unit of oil. This ratio lets you get an idea of how much one resource can substitute for the other when your aim is to produce solid fuel at a large scale. 
```
```suggestion
We find that one unit of coal holds roughly eight times the energy as one unit of oil. This ratio lets you get an idea of how much one resource can substitute for the other when your aim is to produce solid fuel at a large scale. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+* **Energy yield per unit:** As we just found out, that coal holds about 8 times as much energy per unit.
+
+* **Transportation efficiency:** 1 cargo wagon holds 2000 coal, which contains about 16.2GJ of energy if converted to solid fuel. The same cargo wagon holds 400 crude oil barrels, or 20 000 oil, which can yield about 20.6GJ of energy (27% more). If you use a fluid wagon instead, you can hold 25 000 oil per wagon or 25.8GJ (nearly 60% more), meaning that oil wagons transport more energy than coal  ...
```
```suggestion
* **Transportation efficiency:** One cargo wagon holds 2000 coal, which contains about 16.2GJ of energy if converted to solid fuel. The same cargo wagon holds 400 crude oil barrels, or 20 000 oil, which can yield about 20.6GJ of energy (27% more). If you use a fluid wagon instead, you can hold 25 000 oil per wagon or 25.8GJ (nearly 60% more), meaning that oil wagons transport more energy than coal wagons, assuming you don't have to worry about transporting water/steam. Oil also has the transportation option of pipelines, which may be preferable to trains in some cases. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+So far we have done our calculations without using any modules for the sake of keeping things simple and understanding the default conditions. Now let's look at them again with modules and see if our results change significantly. Modules primarily affect the power usage of the fuel processing machines. Let's first review the equation:
+
+`Total processing energy (E) = Machine power usage (P) * time of 1 crafting cycle (t_c) * numnber of crafting cycles (n) / crafting speed (C)`
```
```suggestion
`Total processing energy (E) = Machine power usage (P) * time of one crafting cycle (t_c) * numnber of crafting cycles (n) / crafting speed (C)`
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+Well, the input and output have the same energy now, but the processing energy has grown big enough to wipe all the fuel energy gains. Looking at the cause of this high power cost, we see that it is because this recipe takes 30 seconds to craft while before we were working with 2 or 5 seconds. This difference means that a higher power consumption will has a much greater impact overall.
+
+However, we have other options for making rocket fuel: We can use assembling machine 3s instead and take advantage of their 4 module slots. Here is how some different combinations perform:
```
```suggestion
However, we have other options for making rocket fuel: We can use assembling machine 3s instead and take advantage of their four module slots. Here is how some different combinations perform:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Alright, so I tried 4 productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the fuel energy it put out! So next I tried to remove a productivity module and dampen some of the inflated power consumption with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed m ...
```
```suggestion
Alright, so I tried four productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the fuel energy it put out! So next I tried to remove a productivity module and dampen some of the inflated power consumption with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed module to restore the speed of the machine and cut down the processing time. While this performed better than the efficiency module, it still did not make enough of a difference. I tried more combinations after that and even tried adding various beacon combinations too (no diagrams for those) but no matter what I only managed to get close to a breakeven. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+However, energy yield per unit is not the only factor when comparing coal and oil for solid fuel production. I've identified these factors: 
+
+* **Energy yield per unit:** As we just found out, that coal holds about 8 times as much energy per unit.
```
```suggestion
* **Energy yield per unit:** As we just found out, that coal holds about eight times as much energy per unit.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+**(2) Coal liquification with cracking:** If we also crack the heavy oil before making the solid fuel, it yields 103% extra energy per coal, when no modules are used.
+
+**(3) Coal vs. oil for solid fuel production:** Coal produces about 8 times as much solid fuel or energy as crude oil per unit, when no modules are used. However, you can transport about 60% more energy in a fluid wagon of oil than a cargo wagon of coal. Meanwhile, coal has many exclusive uses it may be needed for instead of making oil products, while crude oil has only one such exclusive use. Th ...
```
```suggestion
**(3) Coal vs. oil for solid fuel production:** Coal produces about eight times as much solid fuel or energy as crude oil per unit, when no modules are used. However, you can transport about 60% more energy in a fluid wagon of oil than a cargo wagon of coal. Meanwhile, coal has many exclusive uses it may be needed for, instead of making oil products, while crude oil has only one such exclusive use. Therefore, using oil for your solid fuel production may be more practical, but it ultimately depends on the abundances of coal and oil on your map.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+`Energy gain = 69 - 40 - 1.5 - 2.1 - 1.365 - 0.84 - 0.21 = 22.985MJ`
+
+`Percent gain = 22.985MJ / 40MJ = 57.4625% `
```
```suggestion
`Percent gain = 22.985 MJ / 40 MJ = 57.4625%`
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+`Energy gain = 88.5 - 40 - 1.5 - 2.1 - 0.6825 - 2.8875 - 0.21 = 41.12MJ`
+
+`Percent gain = 41.12MJ / 40MJ = 102.8% `
```
```suggestion
`Percent gain = 41.12 MJ / 40 MJ = 102.8%`
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+Alright, so I tried 4 productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the fuel energy it put out! So next I tried to remove a productivity module and dampen some of the inflated power consumption with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed m ...
+
+Based on these examples, it seems that while modules do offer some significant energy gain boosts, they do not necessarily make every single fuel processing step better (it depends on the recipe crafting time vs. the energy gains).
```
```suggestion
Based on these examples, it seems that while modules do offer some significant boosts to energy gains, they do not necessarily make every single fuel processing step better (it depends on the recipe crafting time vs. the energy gains).
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses. 
+- **Gains in energy density (per unit and per stack):**  More energy per stack means the same inventory slot can hold more energy, which makes a significant difference for how much energy you can fit into each cargo wagon or chest. Meanwhile, more energy per unit means more energy can be carried by robots, belts, and inserters, which can only hold a few units at a time. If your energy consumption ...
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock cost of the required technology. Each tier of processing adds to these costs.
+- **Availability of more coal/oil to just use instead:** Since these resources are technically infinite on most maps, you may prefer to skip the energy gains and brute force your fuel shortages by setting up more resource extraction infrastructure instead of more fuel processing infrastructure. This factor simply depends on your surroundings.
```
```suggestion
- **Availability of more coal/oil to just use instead:** Since these resources are practically infinite on most maps, you may prefer to skip the energy gains and brute-force your way out of fuel shortages by setting up more resource extraction infrastructure, instead of building more fuel processing infrastructure. This factor simply depends on your surroundings.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+* **Gains in energy:** If you get more energy out of the processed fuel, then your demand for the unprocessed fuel decreases and so your resource patches run out more slowly. We determined the energy gains for different fuels earlier.
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses. 
+- **Gains in energy density (per unit and per stack):**  More energy per stack means the same inventory slot can hold more energy, which makes a significant difference for how much energy you can fit into each cargo wagon or chest. Meanwhile, more energy per unit means more energy can be carried by robots, belts, and inserters, which can only hold a few units at a time. If your energy consumption ...
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock cost of the required technology. Each tier of processing adds to these costs.
```
```suggestion
- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock costs of the required technologies. Each tier of processing adds to these costs.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+Since the above factors depend on your map and your plans, the conclusion varies on whether fuel processing is useful. Based on my experiences, I can offer the following advice about processing fuels:
+
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%) and nuclear fuel is on another level... (+150%)
```
```suggestion
- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%), and nuclear fuel is on another level... (+150%)
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+Since the above factors depend on your map and your plans, the conclusion varies on whether fuel processing is useful. Based on my experiences, I can offer the following advice about processing fuels:
+
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%) and nuclear fuel is on another level... (+150%)
+- **You already need to mass produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
```
```suggestion
- **You already need to mass-produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%) and nuclear fuel is on another level... (+150%)
+- **You already need to mass produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
+- **If you are burning coal for power, processing it into solid fuel will cut the coal demand by half.** And of course, you can go all out and run your boilers with nuclear fuel to reduce the demand to about 1/20th. However, unlocking coal liquification or nuclear fuel requires production science, which means you can unlock solar and (regular) nuclear power much sooner and [their advantages](http ...
```
```suggestion
- **If you are burning coal for power, processing it into solid fuel will cut the coal demand by half.** And of course, you can go all out and run your boilers with nuclear fuel to reduce the coal demand by a factor of 20. However, unlocking coal liquification or nuclear fuel requires production science, which means you can unlock solar and (regular) nuclear power much sooner, and [their advantages](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) may make it worthwhile to switch to them before upgrading your coal setup. Nuclear power is [easier to do than it may seem]([Alt-F4 #57 - Nuclear Energy and You](https://alt-f4.blog/ALTF4-57/#)), as it provides massive energy yields for very few reactors and centrifuges, without needing Kovarex Enrichment!
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+**(5) Rocket fuel vs. nuclear fuel:** Converting rocket fuel into nuclear fuel yields around 10x as much energy as the initial solid fuel that went in, when no modules are used. This makes it the best way to get the most energy out of your initial coal or oil. However, it is not the most efficient way to generate power from uranium.
+
+**(6) Nuclear fuel vs. Uranium fuel cells:** Using uranium-235 for nuclear reactor fuel cells yields up to hundreds of times more energy from it compared to using it for nuclear fuel but you unlock nuclear fuel alongside Kovarex enrichment, meaning that uranium-235 stops being rare when nuclear fuel is available.
```
```suggestion
**(6) Nuclear fuel vs. Uranium fuel cells:** Using uranium-235 for nuclear reactor fuel cells yields up to hundreds of times more energy from it compared to using it for nuclear fuel, but you unlock nuclear fuel alongside Kovarex enrichment, meaning that uranium-235 stops being rare when nuclear fuel is available.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+**(6) Nuclear fuel vs. Uranium fuel cells:** Using uranium-235 for nuclear reactor fuel cells yields up to hundreds of times more energy from it compared to using it for nuclear fuel but you unlock nuclear fuel alongside Kovarex enrichment, meaning that uranium-235 stops being rare when nuclear fuel is available.
+
+**(7) Adding modules for energy gains:** Efficiency modules offer energy gains by reducing the processing power usage. Productivity modules offer energy gains by creating more processed fuel. Speed modules might enhance productivity module effects by compensating for speed losses. Modules in general are more effective in boosting the gains for recipes with shorter crafting times and higher energy ...
```
```suggestion
**(7) Adding modules for energy gains:** Efficiency modules offer energy gains by reducing the processing energy usage. Productivity modules offer energy gains by creating more processed fuel. Speed modules might enhance productivity module effects by compensating for speed losses. Modules in general are more effective in boosting the gains for recipes with shorter crafting times and higher energy gains.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,367 @@
+
+**(7) Adding modules for energy gains:** Efficiency modules offer energy gains by reducing the processing power usage. Productivity modules offer energy gains by creating more processed fuel. Speed modules might enhance productivity module effects by compensating for speed losses. Modules in general are more effective in boosting the gains for recipes with shorter crafting times and higher energy ...
+
+**(8) Usefulness of processing fuels:** It depends on a number of factors but ultimately you do it anyway in order to make rocket parts and it is really useful for vehicles at least.
```
```suggestion
**(8) Usefulness of processing fuels:** It depends on a number of factors, but ultimately you have to make rocket fuel anyway in order to make rocket parts, and until you get to that stage it is really useful for vehicles.
```

LevFendi committed:
Apply suggestions from code review

Thank you Trez!

Co-authored-by: MyNameIsTrez <32989873+MyNameIsTrez@users.noreply.github.com>

LevFendi committed:
Edits:

-"MJ" spacing

-Other details discussed in #writing-feedback

-Rewrote most of modules section and some of the summary

LevFendi committed:
Small additions

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+`Percent gain = 45.824 MJ / 40 MJ = 114.56% ≈ 115%` 
+
+The energy gain was 103% with no modules. We see that efficiency modules increase the energy gain from 103% to 115% This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Nevertheless, efficiency 1 modules being cheap to produce makes this option worthwhile. 
```
```suggestion
The energy gain was 103% with no modules. We see that efficiency modules increase the energy gain from 103% to 115%. This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Nevertheless, efficiency 1 modules being cheap to produce makes this option worthwhile. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+The energy gain was 103% with no modules. We see that efficiency modules increase the energy gain from 103% to 115% This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Nevertheless, efficiency 1 modules being cheap to produce makes this option worthwhile. 
+
+Now, let us look at the same example with three productivity 3 modules in every machine instead of efficiency 1 modules. Despite increasing the power consumption they also increase the amount of fuel produced, by 10% per machine per module:
```
```suggestion
Now, let us look at the same example with three productivity 3 modules in every machine, instead of three efficiency 1 modules. Despite increasing the power consumption, each module also increases the amount of fuel produced by 10%:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100 MJ! However, we also see that total power consumption has greatly increased to about 50.6 MJ.  Let's see how all these changes reflect to the energy gain: 
```
```suggestion
We see that with the cumulative effects of 30% extra product output across every step in the process, the amount of solid fuel is doubled! This adds up to an extra energy gain of nearly 100 MJ! However, we also see that total power consumption has greatly increased to about 50.6 MJ. Let's see how all these changes reflect to the energy gain:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+`Percent gain = 93.3804 MJ / 40 MJ = 233.451% ≈ 233%`
+
+We find that productivity modules increase energy gains from 103% to 233%, meaning that in this setup the solid fuel being produced has **over triple the energy** as the coal used! 
```
```suggestion
We find that productivity modules increase energy gains from 103% to 233%, meaning that in this setup the produced solid fuel has **over three times as much energy** as the coal that was used for it! 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at high productivity & speed'
+ caption='Processing coal into solid fuel with heavy oil cracking, , at high productivity & speed'
```
```suggestion
 caption='Processing coal into solid fuel with heavy oil cracking, at high productivity & speed'
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+%}
+
+Compared to the last setup, we now get less than double the solid fuel but the power consumption has approximately halved. Let's see how it reflects to the energy gain: 
```
```suggestion
Compared to the last setup, we now get less than double the solid fuel, but the power consumption has approximately halved. Let's see how it reflects to the energy gain: 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+`Percent gain = 84.6834 MJ / 40 MJ = 211.7085% ≈ 212%`
+
+Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweigh the reduced power consumption from the speed 3 module. 
```
```suggestion
Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweighs the reduced power consumption from the speed 3 module.
```

LevFendi replied:
Upon discussion, I decided to keep it unchanged.

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweigh the reduced power consumption from the speed 3 module. 
+
+Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumptions to account for. I estimate that good boosts in energy gains are possible by having each beacon affect multiple machine ...
```
```suggestion
Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumption that would have to be accounted for. I predict that another energy gain is possible by having each beacon affect multiple machines. However, since calculations involving beacons would introduce additional factors that would have to be considered, such as machine counts and the physical layout of the fuel processing area, I'll leave that rabbit hole as an exercise for the reader.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumptions to account for. I estimate that good boosts in energy gains are possible by having each beacon affect multiple machine ...
+
+Meanwhile, seeing the strong effects of productivity modules, I wonder if we can use them to make rocket fuel production yield net energy gains. So, let's fill the assembling machine 2 with productivity 3 modules:
```
```suggestion
Meanwhile, seeing the strong effects of productivity modules, I wonder if we can use them to make rocket fuel production yield another net energy gain. So, let's fill the assembling machine 2 with productivity 3 modules:
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+%}
+
+So, I tried four productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the total fuel energy it put out! Next, I tried to dampen some of the inflated power consumption and speed loss by replacing the fourth productivity 3 module with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed  ...
```
```suggestion
So, I tried four productivity 3 modules in an assembling machine 3, but this slowed down the machine so much that the processing energy was almost more than the total fuel energy it put out! Next, I tried to dampen some of the inflated power consumption and speed loss by replacing the fourth productivity 3 module with an efficiency 3 module. This was helpful, but insufficient. Afterwards, I changed strategy and used a speed 3 module instead of the efficiency 3 module, to restore the speed of the machine and to cut down the processing time. While this saved more energy than the efficiency module, it still did not make enough of a difference. I tried more module combinations after that and even tried adding various beacon combinations too (no diagrams for those), but no matter what, I only managed to get close to a breakeven. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+So, I tried four productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the total fuel energy it put out! Next, I tried to dampen some of the inflated power consumption and speed loss by replacing the fourth productivity 3 module with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed  ...
+
+Based on these examples, it seems that while modules do offer some significant boosts to energy gains, they do not necessarily make every single fuel processing step offer net gains.
```
```suggestion
Based on these examples, it seems that while modules can offer a significant increase in the energy gain for some fuel processing steps, it isn't the case for every step.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses as seen in [the wiki page for fuels](https://wiki.factorio.com/Fuel).
+- **Gains in energy density (per unit and per stack):**  More energy per stack means the same inventory slot can hold more energy, which makes a significant difference for how much energy you can fit into each cargo wagon or chest. Meanwhile, more energy per unit means more energy can be carried by robots, belts, and inserters, which can only hold a few units at a time. If your energy consumption ...
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock costs of the required technologies. Each tier of processing adds to these costs.
+- **Availability of more coal/oil to just use instead:** On most maps, coal and oil are practically infinite, but you may not have any new patches nearby. Depending on your preferences, it may be easier to head out and set up more resource extraction infrastructure rather than building more fuel processing infrastructure.
```
```suggestion
- **Availability of more coal/oil to just use instead:** On most maps, coal and oil are practically infinite, but you may not have any new patches nearby. Depending on your preferences, it may be easier to head out and set up more resource extraction infrastructure, rather than building more fuel processing infrastructure.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%), and nuclear fuel is on another level... (+150%)
+- **You already need to mass-produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
+- **If you are burning coal for power, processing it into solid fuel can cut the coal demand by half,** or more if you use modules. And of course, you can go all out and run your boilers with nuclear fuel to reduce the coal demand by a factor of 20. 
```
```suggestion
- **If you are burning coal for power, processing it into solid fuel can cut the coal demand by half,** or more if you use modules. And, of course, you can go all out and run your boilers with nuclear fuel to reduce the coal demand by a factor of 20. 
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%), and nuclear fuel is on another level... (+150%)
+- **You already need to mass-produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
+- **If you are burning coal for power, processing it into solid fuel can cut the coal demand by half,** or more if you use modules. And of course, you can go all out and run your boilers with nuclear fuel to reduce the coal demand by a factor of 20. 
+- **However, you can unlock solar and (regular) nuclear power much sooner than coal liquification and nuclear fuel,** because these two fuel processing technologies require production science to unlock. Meanwhile, [the advantages of solar and nuclear power](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) may make it worthwhile to switc ...
```
```suggestion
- **However, you can unlock solar and (regular) nuclear power much sooner than coal liquification and nuclear fuel,** because these two fuel processing technologies require production science to unlock. Meanwhile, [the advantages of solar and nuclear power](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) may make it worthwhile to switch to them, instead of waiting to upgrade your coal power setup. Nuclear power is [easier to do than it may seem]([Alt-F4 #57 - Nuclear Energy and You](https://alt-f4.blog/ALTF4-57/#)), as it provides massive energy yields for very few reactors and centrifuges, without needing Kovarex Enrichment!
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 57% extra energy per coal, when no modules are used. 
+
+**(2) Coal liquification with cracking:** If we also crack the heavy oil before making the solid fuel, it yields 103% extra energy per coal, when no modules are used. Hence we get about twice the energy from the same coal.
```
```suggestion
**(2) Coal liquification with cracking:** If we also crack the heavy oil before making the solid fuel, it yields 103% extra energy per coal, when no modules are used. Hence, we get about twice the energy from the same coal.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules, because they are quite cheap to make and they cut down power usage by 60% or even 80% (the eff ...
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation, reduces the power usage because the added speed makes the machine work for a shorter time for each crafti ...
```
```suggestion
- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation reduces the power usage, because the added speed makes the machines work for a shorter time for each crafting cycle. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which diminishes the overall energy gain. However, speed modules can be used alongside productivity modules to mitigate the slowdown they cause, and this might provide energy gains overall.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+**(8) Usefulness of processing fuels:** It depends on a number of factors, but ultimately you have to make rocket fuel anyway in order to make rocket parts, and even before then processed fuels are useful for vehicles.
+
+These questions have been pretty fun for me to explore and I hope they were interesting to you to read! Do you have any additions or similar questions about fuels or other game mechanics? Feel free to get in touch on Reddit or Discord via the links at the bottom of the article!
```
```suggestion
These questions have been pretty fun for me to explore and I hope they were interesting for you to read! Do you have any additions or similar questions about fuels or other game mechanics? Feel free to get in touch on Reddit or Discord via the links at the bottom of the article!
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+This article started with a little burning question that brought more such questions and evolved into an exploration of all of Factorio's fuels. We uncovered the following:
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 57% extra energy per coal, when no modules are used. 
```
```suggestion
**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 57% extra energy per coal, when no modules are used.
```

MyNameIsTrez commented on submissions/burning-questions/burning-questions.md:
```diff
@@ -0,0 +1,382 @@
+
+**(6) Nuclear fuel vs. Uranium fuel cells:** Using uranium-235 for nuclear reactor fuel cells yields up to hundreds of times more energy from it compared to using it for nuclear fuel, but you unlock nuclear fuel alongside Kovarex enrichment, meaning that uranium-235 stops being rare when nuclear fuel is available.
+
+**(7) Adding modules for energy gains:** Efficiency modules offer energy gains by reducing the processing energy usage, and efficiency 1 modules make a good difference while being cheap to produce. Productivity modules can offer energy gains by creating more processed fuel, and the effect compounds such that you can get exponential increases in the fuel output overall. Speed modules can offer ene ...
```
```suggestion
**(7) Adding modules for energy gains:** Efficiency modules offer energy gains by reducing the processing energy usage, and efficiency 1 modules make a good difference while being cheap to produce. Productivity modules can offer energy gains by creating more processed fuel, and the effect compounds such that it leads to exponential increases overall in the fuel output. Speed modules can offer energy gains when used alongside productivity modules by compensating for the speed losses that increase the power consumption. 
```

LevFendi committed:
Apply suggestions from code review

Many thanks again to Trez!

Co-authored-by: MyNameIsTrez <32989873+MyNameIsTrez@users.noreply.github.com>

LevFendi committed:
"a net energy gain as well"

LevFendi committed:
Links to modules articles

LevFendi committed:
Tidying, better modules overview visual

LevFendi committed:
Visuals: Export to PNG, rename, reference in text

Conor12345 committed:
Tech edit
